Tweeter.Tests/DAL/TweeterRepoTests.cs

```diff
@@ -109,6 +136,52 @@ public void RepoEnsureUsernameExistsOfTwit()
 
             // Assert
             Assert.IsNotNull(found_twit);
+    }
+        [TestMethod]
+        public void RepoAddTweet()
+        {
+            ConnectToDatastore();
+            Tweet newTweet = new Tweet { TweetId = 3, Message = "hello there" };
+
+            Repo.AddTweet(newTweet);
+            int expected_num_of_tweets = 3;
+            int actual_num_of_tweets = tweets.Count();
+
+            Assert.AreEqual(expected_num_of_tweets, actual_num_of_tweets);
+
+        }
+        [TestMethod]
+        public void RepoRemoveTweet()
+        {
+            ConnectToDatastore();
+            Tweet newTweet = new Tweet { TweetId = 3, Message = "hello there" };
+            Repo.AddTweet(newTweet);
+
+            Repo.RemoveTweet(3);
```

Comment:
You can also remove `AddTweet` and just remove a `Tweet` [already seeded](https://github.com/nss-evening-cohort-3/tweeter/pull/51/files#diff-d52f97db2404fd0c77b2c9142f6a8150R42) in your datastore.